### PR TITLE
ALC-135 navigation and logo

### PR DESCRIPTION
**Pull request by tomheather-50**
Responsive navigation with mistX logo and external icon links as per figma designs 

- some design tweaks could be made for mobile but suggest this is tackled in a later task

### CODE REVIEW

**Comment by tomheather-50** [comment]
will update this PR for light mode

**Comment by Dire-0x** [comment]
Looking great, almost there. A few things..

- There should be more padding between the nav items in the left nav. 
- The coingecko icon appears bigger than the others.
- Make sure the yellow circle highlight around the icon has the right amount of padding. ( This might be corrected when you fix the coingecko icon size )

File: package.json [hunk 1]
```diff
@@ -100,7 +100,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "integration-test": "start-server-and-test 'serve build -l 3000' http://localhost:3000 'cypress run'"
+    "integration-test": "start-server-and-test 'serve build -l 3003' http://localhost:3003 'cypress run'"
```
Comment (Dire-0x) [inline package.json]: Let's revert this

**Review by Dire-0x — APPROVED** [review]
Looks good, lets get this merged!

**Pull request merged**